discord_bot.js

```diff
@@ -1,49 +1,46 @@
-var fs = require('fs');
-var _ = require('lodash');
+const fs = require('fs');
+const _ = require('lodash');
 
-process.on('unhandledRejection', (reason) => {
-  console.error(reason);
-  process.exit(1);
-});
+// not needed if you don't have errors
+//-------------------------------------------------
+//process.on('unhandledRejection', (reason) => {
+//  console.error(reason);
+//  process.exit(1);
+//});
```

Comment:
We've had various errors pop up in different places, and the default behavior of just ignoring and continuing made it quite difficult to track them down.  I don't think we want to remove this.

Reply:
I'm far from an expert on node however (my day job is C++); has something changed in more recent versions of node so that the behavior for unhandled rejections is more sane now?

Reply:
Sure. No problem.